### DISCUSSION TITLE
Password disable hangs

### DISCUSSION THREAD

**Discussion by Battum** [rern/rAudio#598] [Bug Reports]
Hello. I successfully installed the latest version and got it up and running.
At first I thought it was a good idea to set up a password, but in practice it's not convenient, so I tried to disable it in Settings, Features, Others. Disabled the Password Login, entered the current password.
There it's stuck.
In the left bottom corner Password Login / Disable is visible with the lock flashing. And does that forever. Refreshing the screen of going to another screen sets the password back in place.

It seems like a bug. Is there another way to get rid of this password or do I have to wait for an update?
![2021-11-09_095346](https://user-images.githubusercontent.com/93975669/140893377-1bd10710-172c-4140-97c5-1c6533d018ab.jpg)

Peter

**Comment by rern** [comment]
Fixed by today's update.
